pkg/report/report.go

```diff
@@ -369,6 +369,8 @@ type stackParams struct {
 	stackStartRes []*regexp.Regexp
 	// frameRes match different formats of lines containing kernel frames (capture function name).
 	frameRes []*regexp.Regexp
+	// frameSkipRes match formats of lines that should be skipped from the stack trace
+	frameSkipRes []*regexp.Regexp
```

Comment:
We have questionableRe for ` ? ` x86 frames, I think the `(unreliable)` on power means the same? If so please add it to the questionableRe.
This logic is already insanely complex and any new concepts have chances of breaking things in other places. E.g. in some places we count number of lines between things, and this frameSkipRes kicks in in a different place as compared to questionableRe so can break line counting.

Reply:
It's sufficiently complex that I didn't even notice `questionableRe`, it seems... will do

Reply:
questionableRe doesn't seem to do what I want it to do. I'm staring at it a bit harder to figure out why.

Reply:
never mind, turns out i just screwed up the regex